pkg/crc/config/secret_config.go

```diff
@@ -20,32 +20,25 @@ func (s Secret) String() string {
 }
 
 type SecretStorage struct {
-	secretService   string
-	storeAccessible bool
+	secretService string
 }
 
 func NewSecretStorage() *SecretStorage {
 	return &SecretStorage{
-		secretService:   secretServiceName,
-		storeAccessible: keyringAccessible(),
+		secretService: secretServiceName,
 	}
 }
 
 func (c *SecretStorage) Get(key string) interface{} {
-	if !c.storeAccessible {
-		return nil
-	}
 	secret, err := keyring.Get(c.secretService, key)
-	if errors.Is(err, keyring.ErrNotFound) {
+	if err != nil || errors.Is(err, keyring.ErrNotFound) {
```

Comment:
The `errors.Is()` check seems to be dead code? It will only be reached when `err == nil`, and a `nil` error won't match `keyring.ErrNotFound`?